manifests/releases/cf-cli.yml

```diff
@@ -2,6 +2,6 @@
   path: /releases?/name=cf-cli?
   value:
     name: cf-cli
-    version: 1.33.0
-    url: https://bosh.io/d/github.com/bosh-packages/cf-cli-release?v=1.33.0
-    sha1: c038fdb8e9c0707dc407bf0f2846ad324a5d8d9b
+    version: 1.36.0
```

Comment:
One of the things this package does is add the cf v6, cf v7, and cf v8 to the release.   How is s the cf command being used in this bosh release?    Are there any tests that might need to be updated using the different versions of the CF command?